src/cbuild/util/cmake.py

```diff
@@ -100,6 +100,23 @@ def configure(
     )
 
 
+def install(pkg, build_dir=None, extra_args=[], env={}):
+    if not build_dir:
+        build_dir = pkg.make_dir
+    renv = {"DESTDIR": pkg.chroot_destdir}
+    env.update(renv)
+
+    pkg.do(
+        *pkg.make_install_wrapper,
+        "cmake",
+        "--install",
+        ".",
+        *extra_args,
+        wrksrc=build_dir,
```

Comment:
it's also possible to do `--install build_dir` instead of passing wrksrc at all, which probably looks better, but i used `.` with wrksrc= for consistency